src/domexpression.js

```diff
@@ -31,6 +34,21 @@ var _ = Mavo.DOMExpression = $.Class({
 			this.expression = expression;
 		}
 
+		if (this.attribute?.startsWith("mv-attr-")) {
+			this.originalAttribute = this.attribute;
+			this.attribute = this.attribute.replace("mv-attr-", "");
+
+			if (["http://www.w3.org/2000/svg", "http://www.w3.org/1998/Math/MathML"].includes(this.element.namespaceURI)) {
```

Comment:
Nitpick but we should move these URLs to constants (e.g. `SVG_NAMESPACE_URI` and `MATHML_NAMESPACE_URI`)

Reply:
Agreed. Done!